src/components/DisplayEmbed/DisplayExternal.tsx

```diff
@@ -112,132 +106,145 @@ export class DisplayExternal extends Component<Props, State> {
-          }
-        }>
-        <FigureButtons
-          language={language}
-          tooltip={t('form.external.remove', {
-            type: providerName || t('form.external.title'),
-          })}
+  const openEditEmbed = (evt: MouseEvent) => {
+    evt.preventDefault();
+    setIsEditMode(true);
+  };
+
+  const closeEditEmbed = () => {
+    setIsEditMode(false);
+  };
+
+  const onEditEmbed = (properties: Embed) => {
+    saveEmbedUpdates(properties);
+    closeEditEmbed();
+  };
+
+  const showCopyOutline = isSelectedForCopy && (!isEditMode || !active);
+
+  return (
+    <>
+      {!properties.src || !properties.type ? (
+        <div />
+      ) : !allowedProvider || error ? (
+        <EditorErrorMessage
           onRemoveClick={onRemoveClick}
-          embed={embed}
-          providerName={providerName}
-          figureType="external"
-          onEdit={
-            allowedProvider.name
-              ? evt => {
-                  evt.preventDefault();
-                  evt.stopPropagation();
-                  this.openEditEmbed(evt);
-                }
-              : undefined
+          msg={
+            error
+              ? t('displayOembed.errorMessage')
+              : t('displayOembed.notSupported', properties.type, properties.provider)
           }
         />
-        {(embed.resource === 'iframe' || embed.resource === 'external') &&
-        embed.type === 'fullscreen' ? (
-          <SlateResourceBox embed={embed} language={language} />
-        ) : (
-          <iframe
-            contentEditable={false}
-            src={src}
-            height={allowedProvider.height || height}
-            title={title}
-            scrolling={type === 'iframe' ? 'no' : undefined}
-            allowFullScreen={true}
-            frameBorder="0"
+      ) : (
+        <div className={'c-figure'}>
+          <FigureButtons
+            language={language}
+            tooltip={t('form.external.remove', {
+              type: providerName || t('form.external.title'),
+            })}
+            onRemoveClick={onRemoveClick}
+            embed={embed}
+            providerName={providerName}
+            figureType="external"
+            onEdit={
+              allowedProvider.name
+                ? evt => {
+                    evt.preventDefault();
+                    evt.stopPropagation();
+                    openEditEmbed(evt);
+                  }
+                : undefined
+            }
           />
-        )}
-        <DisplayExternalModal
-          embed={embed}
-          isEditMode={isEditMode}
-          src={src}
-          type={type}
-          onEditEmbed={this.onEditEmbed}
-          onClose={this.closeEditEmbed}
-          allowedProvider={allowedProvider}
-        />
-      </div>
-    );
-  }
-}
+          {(embed.resource === 'iframe' || embed.resource === 'external') &&
+          embed.type === 'fullscreen' ? (
+            <SlateResourceBox embed={embed} language={language} />
```

Comment:
Nå som boxShadow ble flyttet ned til diven under så vil ikke den gjelde for scenarioet når embed.type === fullscreen. Kan eventuelt wrappe denne komponenten i en div med egen boxShadow.